bundles/org.openhab.persistence.jpa/src/main/java/org/openhab/persistence/jpa/internal/JpaPersistenceService.java

```diff
@@ -164,7 +165,13 @@ public void store(Item item, @Nullable String alias) {
             em.getTransaction().commit();
             logger.debug("Persisting item...done");
         } catch (Exception e) {
-            logger.error("Error while persisting item! Rolling back!", e);
+            if (e.getCause() instanceof EntityExistsException) {
+                // there's a UNIQUE constraint in the database, and we tried to write
+                // a duplicate timestamp. Just ignore
+                logger.debug("Skipping persistence of item... it already exists");
```

Comment:
For this to be of much use, I think we need to either add the item name, or include the call stack and hope the exception has enough details. E.g.:
```suggestion
                logger.debug("Failed to persist item {} because of duplicate timestamp", name);
```